.travis.yml

```diff
@@ -16,146 +16,33 @@ cache:
-  - |
-    if [ "$TRAVIS_OS_NAME" == "linux" ]; then
-      # adding apt repos in travis is really fragile => retry a couple of times.
-      for i in {1..10}; do travis_retry sudo apt-add-repository --yes 'ppa:ubuntu-toolchain-r/test' && break; sleep 2; done
-      for i in {1..10}; do travis_retry sudo apt-get update && travis_retry sudo apt-get install --yes gcc-8  $(if [ -z "${BACKEND##*cpp*}" ]; then echo -n "g++-8"; fi ) && break; sleep 2; done
-      sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-8 60 $(if [ -z "${BACKEND##*cpp*}" ]; then echo " --slave /usr/bin/g++ g++ /usr/bin/g++-8"; fi)
-      sudo update-alternatives --set gcc /usr/bin/gcc-8
-      export CC=gcc
-      if [ -z "${BACKEND##*cpp*}" ]; then sudo update-alternatives --set g++ /usr/bin/g++-8; export CXX=g++; fi
-    fi
-
-  - |
-    if [ "$TRAVIS_OS_NAME" == "osx" -o "$STACKLESS" == "true" ]; then
-      echo "Installing Miniconda"
-      if [ "$TRAVIS_OS_NAME" == "osx" ]; then CONDA_PLATFORM=MacOSX; else CONDA_PLATFORM=Linux; fi
-      travis_retry wget -O miniconda.sh https://repo.continuum.io/miniconda/Miniconda$PY-latest-${CONDA_PLATFORM}-x86_64.sh || exit 1
-      bash miniconda.sh -b -p $HOME/miniconda && rm miniconda.sh || exit 1
-      conda --version || exit 1
-      #conda install --quiet --yes nomkl --file=test-requirements.txt --file=test-requirements-cpython.txt
-      if [ "$TRAVIS_OS_NAME" == "osx" ]; then
-        which clang && clang --version && export CC="clang -Wno-deprecated-declarations" || true
-        which clang++ && clang++ --version && export CXX="clang++ -stdlib=libc++ -Wno-deprecated-declarations" || true
-      fi
-    fi
-
-  - if [ -n "$CC" ]; then which ${CC%% *}; $CC --version; fi
-  - if [ -n "$CXX" ]; then which ${CXX%% *}; $CXX --version; fi
-
-  - if [ "$STACKLESS" == "true" ]; then
-      conda config --add channels stackless;
-      travis_retry conda install --quiet --yes stackless;
-    fi
-
-install:
-  - python -c 'import sys; print("Python %s" % (sys.version,))'
-  - if [ -n "${TRAVIS_PYTHON_VERSION##*-dev}" ]; then pip install -r test-requirements.txt $( [ -z "${TRAVIS_PYTHON_VERSION##pypy*}" -o -z "${TRAVIS_PYTHON_VERSION##3.[4789]*}" ] || echo " -r test-requirements-cpython.txt" ) ; fi
-#  - CFLAGS="-O2 -ggdb -Wall -Wextra $(python -c 'import sys; print("-fno-strict-aliasing" if sys.version_info[0] == 2 else "")')" python setup.py build
-
-before_script: ccache -s || true
+    # Disabled due to test errors
+    # - arch: s390x
+    #   env: BACKEND=c
+    # - arch: s390x
+    #   env: BACKEND=cpp
 
 script:
-  - PYTHON_DBG="python$( python -c 'import sys; print("%d.%d" % sys.version_info[:2])' )-dbg"
-  - if [ "$TEST_CODE_STYLE" = "1" ]; then
-      STYLE_ARGS="--no-unit --no-doctest --no-file --no-pyregr --no-examples";
-    else
-      STYLE_ARGS=--no-code-style;
-      if $PYTHON_DBG -V >&2; then CFLAGS="-O0 -ggdb" $PYTHON_DBG runtests.py -vv --no-code-style Debugger --backends=$BACKEND; fi;
```

Comment:
This line went missing, I think.

Reply:
Yep, forgot to add it. Thanks for pointing it out